README.md

```diff
@@ -247,6 +247,7 @@ Apache 2 Licensed. See [LICENSE](https://github.com/terraform-aws-modules/terraf
 | <a name="input_enable_irsa"></a> [enable\_irsa](#input\_enable\_irsa) | Whether to create OpenID Connect Provider for EKS to enable IRSA | `bool` | `false` | no |
 | <a name="input_fargate_pod_execution_role_name"></a> [fargate\_pod\_execution\_role\_name](#input\_fargate\_pod\_execution\_role\_name) | The IAM Role that provides permissions for the EKS Fargate Profile. | `string` | `null` | no |
 | <a name="input_fargate_profiles"></a> [fargate\_profiles](#input\_fargate\_profiles) | Fargate profiles to create. See `fargate_profile` keys section in fargate submodule's README.md for more details | `any` | `{}` | no |
+| <a name="input_fargate_subnets"></a> [fargate\_subnets](#input\_fargate\_subnets) | Subnets to use when creating fargate profiles. If not set then the all subnets in the cluster will be used. | `list(string)` | `[]` | no |
```

Comment:
this should be generated by `terraform-docs` basing on input description in line https://github.com/terraform-aws-modules/terraform-aws-eks/blob/4787e3cd390350fb3b14e557c7186aeb884ad736/variables.tf#L91 but seems you have modified this manually.

terraform-docs are triggered in `pre-commit` webhook